subprojects/core/src/main/java/org/gradle/initialization/DefaultGradleLauncher.java

```diff
@@ -105,23 +106,28 @@ public BuildResult load() throws ReportedException {
         return doBuild(Stage.Load);
     }
 
-    private BuildResult doBuild(final Stage upTo) {
-        Throwable failure = null;
+    private BuildResult doBuild(Stage upTo) {
+        BuildOperationWorkerRegistry.Completion workerLease = buildServices.get(BuildOperationWorkerRegistry.class).operationStart();
```

Comment:
Right, but `BuildOperationWorkerRegistry` is still build scoped and this is the earliest we can get it. It is planned to move it to the gradle user home scope soon, we'll move this then.